code-challenges.test.js

```diff
@@ -1,51 +1,158 @@
-// ASSESSMENT 6: JavaScript Coding Practical Questions with Jest
-
-// Please read all questions thoroughly
-// Pseudo coding is REQUIRED
-// If you get stuck, please leave comments to help us understand your thought process
-
-// Use test driven development to complete the following questions
-// Add appropriate dependencies: $ yarn add jest
-
-// Reminder: The test will call your function
-// Run the file with the following command: $ yarn jest
-
-// --------------------1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
-
-// a) Create a test with an expect statement using the variable provided.
-
-const people = [
-  { name: "ford prefect", occupation: "a hitchhiker" },
-  { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "a radio employee" }
-]
-// Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
-
-
-// b) Create the function that makes the test pass.
-
+// // ASSESSMENT 6: JavaScript Coding Practical Questions with Jest=
+// // --------------------1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
+// // // a) Create a test with an expect statement using the variable provided.
+describe("objectArray", () => { 
+    it("Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized", () => {
+    const people = [
+      { name: "ford prefect", occupation: "a hitchhiker" },
+      { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
+      { name: "arthur dent", occupation: "a radio employee" }
+    ]
+// // //     // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
```

Comment:
Keep in mind your expected output is given to you here and should be used in your .toEqual.  No need to run three expect statements.